dts/bindings/input/ilitek,ili2132a.yaml

```
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Joel Jaldemark
+# SPDX-License-Identifier: Apache-2.0
+
+description: ILI2143A capactitive touch controller
```

```suggestion
description: ILI2132A capacitive touch controller
```

That's a weird typo though, so please confirm the driver is indeed meant to be for 2132 and not in fact 2143 :)

---

"capactitive" typo is still there :)